guides/source/active_record_validations.md

```diff
@@ -278,39 +310,149 @@ end
+In modern Rails applications, the more concise validate syntax is commonly used,
+for example:
+
+```ruby
+validate :name, presence: true
+```
+
+However, older versions of Rails used "helper" methods, such as:
 
-Active Record offers many pre-defined validation helpers that you can use
-directly inside your class definitions. These helpers provide common validation
-rules. Every time a validation fails, an error is added to the object's `errors`
-collection, and this is associated with the attribute being validated.
+```ruby
+validates_presence_of :name
+```
+
+Both notations perform the same function, but the newer form is recommended for
+its readability and alignment with Rails' conventions.
 
-Each helper accepts an arbitrary number of attribute names, so with a single
-line of code you can add the same kind of validation to several attributes.
+Each validation accepts an arbitrary number of attribute names, allowing you to
+apply the same type of validation to multiple attributes in a single line of
+code.
 
-All of them accept the `:on` and `:message` options, which define when the
-validation should be run and what message should be added to the `errors`
-collection if it fails, respectively. The `:on` option takes one of the values
-`:create` or `:update`. There is a default error message for each one of the
-validation helpers. These messages are used when the `:message` option isn't
-specified. Let's take a look at each one of the available helpers.
+Additionally, all validations accept the `:on` and `:message` options. The `:on`
+option specifies when the validation should be triggered, with possible values
+being `:create` or `:update`. The `:message` option allows you to define a
+custom error message that will be added to the errors collection if the
+validation fails. If you do not specify a message, Rails will use a default
+error message for that validation.
 
 INFO: To see a list of the available default helpers, take a look at
-[`ActiveModel::Validations::HelperMethods`][].
+[`ActiveModel::Validations::HelperMethods`][]. This API section uses the older
+notation as described above.
+
+[`ActiveModel::Validations::HelperMethods`]:
+    https://api.rubyonrails.org/classes/ActiveModel/Validations/HelperMethods.html
+
+Below we outline the most commonly used validations.
+
+### `absence`
+
+This helper validates that the specified attributes are absent. It uses the
+[`Object#present?`][] method to check if the value is neither nil nor a blank
+string - that is, a string that is either empty or consists of whitespace only.
+
+```ruby
+class Person < ApplicationRecord
+  validates :name, :login, :email, absence: true
+end
+```
+
+If you want to be sure that an association is absent, you'll need to test
+whether the associated object itself is absent, and not the foreign key used to
+map the association. Rails will usually infer the inverse association automatically.
+
+```ruby
+class LineItem < ApplicationRecord
+  belongs_to :order
```

Comment:
Or maybe replace the example with a `has_one` instead of the `belongs_to` as a `has_one` isn't required by default?

```ruby
class Supplier < ApplicationRecord
  has_one :account
  validates :account, presence: true
end

class Account < ApplicationRecord
  belongs_to :supplier
end
```